handler/handler.go

```diff
@@ -244,6 +326,13 @@ func validateUUID(u string) error {
 	return err
 }
 
+func validatePredicate(pr string) error {
```

Comment:
We shouldn't put restriction only on `abouts` and `mentions`. For example someone may want to use mini Mum for changing annotation with predicate `isClassifiedBy`.

Reply:
All the possible predicates are these:
```
http://www.ft.com/ontology/annotation/mentions
http://www.ft.com/ontology/annotation/about
http://www.ft.com/ontology/annotation/hasAuthor
http://www.ft.com/ontology/hasContributor
http://www.ft.com/ontology/hasDisplayTag
http://www.ft.com/ontology/classification/isClassifiedBy
```